trust/check-format.c

```diff
@@ -39,10 +39,12 @@
 #include "check-format.h"
 #include "debug.h"
 #include "message.h"
+#include "persist.h"
 #include "tool.h"
 
 #include <assert.h>
 #include <ctype.h>
+#include <errno.h>
```

Comment:
I think you can remove some includes, like string.h

Reply:
Done; that also reminds me that we should provide a stub for `isatty` on non-Unix platforms, fixed that as well.